src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -189,17 +189,32 @@
 
                     requestStack.push(stackElement);
 
-                    this.addEventListener("readystatechange", function() {
-                        if (self.readyState == 4) {
-                            stackElement.duration = new Date() - stackElement.start;
-                            stackElement.loading = false;
-                            stackElement.error = self.status < 200 || self.status >= 400;
-                            stackElement.profile = self.getResponseHeader("X-Debug-Token");
-                            stackElement.profilerUrl = self.getResponseHeader("X-Debug-Token-Link");
-
-                            Sfjs.renderAjaxRequests();
-                        }
-                    }, false);
+                    /* check for older browsers */
+                    if (!window.addEventListener) {
+                        this.attachEvent("readystatechange", function () {
```

Comment:
It's onreadystatechange if I'm not wrong?
I'd also deduplicate the closure by putting it in a variable